src/views/clusteroverview/SettingsTab/ClusterTab/components/PersistentReservationSection/PersistentReservationSection.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FC, useState } from 'react';
+
+import HyperConvergedModel from '@kubevirt-ui/kubevirt-api/console/models/HyperConvergedModel';
+import HelpTextIcon from '@kubevirt-utils/components/HelpTextIcon/HelpTextIcon';
+import { FEATURE_HCO_PERSISTENT_RESERVATION } from '@kubevirt-utils/hooks/useFeatures/constants';
+import { useFeatures } from '@kubevirt-utils/hooks/useFeatures/useFeatures';
+import { HyperConverged } from '@kubevirt-utils/hooks/useHyperConvergeConfiguration';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { k8sPatch } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Alert,
+  AlertVariant,
+  PopoverPosition,
+  Split,
+  SplitItem,
+  Switch,
+} from '@patternfly/react-core';
+
+import ExpandSection from '../../../ExpandSection/ExpandSection';
+
+type PersistentReservationSectionProps = {
+  hyperConvergeConfiguration: [hyperConvergeConfig: HyperConverged, loaded: boolean, error: Error];
+};
+
+const PersistentReservationSection: FC<PersistentReservationSectionProps> = ({
+  hyperConvergeConfiguration,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const [hyperConverge, hyperLoaded] = hyperConvergeConfiguration;
+  const persistentReservation = hyperConverge?.spec?.featureGates?.persistentReservation ?? false;
```

Comment:
Hmm, do we really need the nullish coalescing operator here? Maybe just 
`!!hyperConverge?.spec?.featureGates?.persistentReservation` would do the same?  Or `||` one instead `??` ?

Reply:
Okay i used `Boolean()`